app/controllers/invite_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class InviteController < ApplicationController
+  def new
+  end
+
+  def create
+    query = github_service.user_query(params[:github_handle])
+    invitee = GitHub::User.new(query)
+    if invitee.email
+      InviteMailer.invite_email(current_user, invitee).deliver_now
+      flash[:success] = 'Successfully sent invite!'
+    else
+      flash[:danger] = "The Github user you selected doesn't have an email address associated with their account."
```

Comment:
Discussed this and landed on this as being the most simple:
Here we query to get an invitee and check if they have a email on GitHub, then uses the mailer to send the email now or displays the message that the invitee doesn't have an email address associated with their GitHub.